src/server.cpp

```diff
@@ -1043,9 +1043,9 @@ void CServer::MixEncodeTransmitData ( const int iChanCnt,
                                       const int iCurChanID,
                                       const int iNumClients )
 {
-    int               i, j, k, iUnused;
-    CVector<double>&  vecdIntermProcBuf = vecvecsIntermediateProcBuf[iChanCnt]; // use reference for faster access
-    CVector<int16_t>& vecsSendData      = vecvecsSendData[iChanCnt];            // use reference for faster access
+    int             i, j, k, iUnused;
+    CVector<float>& vecdIntermProcBuf = vecvecsIntermediateProcBuf[iChanCnt]; // use reference for faster access
```

Comment:
Here, e.g., "vecd" stands for vector double. It should be "vecf" now.

Reply:
Fixed.